assets/stylesheets/_colors.scss

```diff
@@ -0,0 +1,4 @@
+:root {
+	--color-link: #984A9C;
```

Comment:
This seems to be unused.

Reply:
it's used to change the colors of the links to purple, like in the main window here:
https://www.dropbox.com/s/tx2km3l519u3ir8/shot-2019-10-11-16-01-22.png?dl=0